pkg/controller/convert.go

```diff
@@ -82,6 +84,7 @@ func CreateResourceDependencyTree() (root *resourcetree.ResourceNode) {
 	identityProviderNode := createNode(delegatedNameserversConfirmedNode, resourcetree.ResourceNodeTypeIdentityManager)
 	kubePromStack := createNode(identityProviderNode, resourcetree.ResourceNodeTypeKubePromStack)
 	createNode(identityProviderNode, resourcetree.ResourceNodeTypeArgoCD)
```

Comment:
IMO: dependencies should reside beneath their parent. In other words, I'd suggest moving ArgoCD and Users up beneath identityprovider (line 85). I think this will improve readability